Data/Aeson.hs

```diff
@@ -77,9 +77,14 @@ module Data.Aeson
     , GFromJSON(..)
     , GToJSON(..)
     , GToEncoding(..)
+    , Zero
```

Comment:
"Need" is a strong word, but if `GFromJSON`/`GToJSON`/`GToEncoding` are exported from `Data.Aeson`, then I feel it makes sense to export `Zero`/`One` too, since it would now be pretty much impossible to write useful type signatures involving the former without using the latter (since you need things like `GToJSON Zero (Rep a)`).